src/checks.ts

```diff
@@ -17,7 +17,10 @@ export async function checkPackageJson(
         }
         return;
     }
-    if (/download/.test(dirPath)) {
+    if (/download/.test(dirPath) &&
```

Comment:
Probably minor but using `.includes` and using a predefined `Set` here might be a bit faster :)